### PR TITLE
feat(badges): rename challenge to badge

### LINKED ISSUE #130: Add badge to challenge
- [ ] Add img to challenge model
- [ ] Send image with user_challenge view (tips, put it in serializer)

### PR DESCRIPTION
**Pull request by olros**
# feat(badges): rename challenge to badge 🤳
## Description

Comments/issues:
closes #130 

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] PR name is descriptive
- [ ] The PR includes the link to the relevant Trello task
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

- [ ] `python manage.py runserver` was run locally witout errors

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

**Review by Zenjjim — COMMENTED** [review]
Ayy gutten har lært seg backend jo. Du kan godt rename dette til badge.   og legge inn permissions at du kun kan se dine egne

File: app/content/models/user_badge.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+from app.util.models import BaseModel, OptionalImage
+
+from app.content.models import User, Badge
+
+class UserBadge(BaseModel):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    badge = models.ForeignKey(Badge, on_delete=models.CASCADE)
+
+    class Meta:
+        verbose_name_plural = 'User Badges'
```
Comment (Eirsteir) [inline app/content/models/user_badge.py:11]: Perhaps a singular verbose name = "User badge" ?

Comment (olros) [reply app/content/models/user_badge.py]: Agree 👍

**Review by Eirsteir — CHANGES_REQUESTED** [review]
I am impressed by your backend skills Olaf and with your rapid completion, well done! ❤️ 

I could not find any faults in your logic, but there are some things which can be simplified and/or improved.

File: app/content/serializers/user.py [hunk 2]
```diff
@@ -47,6 +49,14 @@ def get_events(self, obj):
         ]
         return EventInUserSerializer(events, many=True).data
 
+    def get_badges(self, obj):
+        """ Lists all badges a user has accomplished """
+        user_badges = UserBadge.objects.filter(user__user_id=obj.user_id).order_by(
```
[the remark this answers is not in the document]
Comment (olros) [reply app/content/serializers/user.py:55]: Tried to do this, but ended up with something like "badges" is not in user. Please help 😅

[the remark this answers is not in the document]
Comment (Eirsteir) [reply app/content/serializers/user.py]: For future reference, the argument is called `related_name` and not `verbose_name` 😅

**Review by Eirsteir — APPROVED** [review]
Nice work 😄 

Please rename your migration-files to something meaningful (remember to include the number prefix) and include the missing migrations file by running `make makemigrations`

**Review by Eirsteir — CHANGES_REQUESTED** [review]
I see now that you have not updated the fixtures, please do so before approval

**Comment by olros** [comment]
Merget branchen `bug/fixtures` inn i denne for å klare å kjøre opp databasen og migrations.

**Review by Eirsteir — APPROVED** [review]
Congrats backender! ❤️

**Review by Zenjjim — CHANGES_REQUESTED** [review]
Før du merger ønsker jeg at du gjør migrations til en fil

**Review by Zenjjim — APPROVED** [review]

**Pull request merged**